src/themes/default/pages/Product.vue

```diff
@@ -143,11 +143,19 @@
                 :name="$t('Quantity')"
                 v-model="product.qty"
                 :min="1"
+                @input="$v.$touch()"
+                :validations="[
+                  {
+                    condition: $v.product.qty.$error && !$v.product.qty.minValue,
+                    text: 'Quantity must be above 0'
```

Comment:
every displayed text have to be with i18n, because we'd lost translation feature